src/v2/Apps/Order/Components/PaymentMethodSummaryItem.tsx

```diff
@@ -24,14 +22,17 @@ export const PaymentMethodSummaryItem = ({
     ...{ textColor: textColor },
   }
   const isACHEnabled = useFeatureFlag("stripe_ACH")
+  const isWireTransferEnabled = useFeatureFlag("wire_transfer")
```

Comment:
I suppose these flags can be removed since we check it on the backend side.

Reply:
agreed. 

they are already removed from `PaymentContent` where we render the radio buttons vs. only Credit card. I left them here for extra safety (not to break anything for the current payment method credit card). 

do you know if we are already getting `paymentMethod === 'CREDIT_CARD'` from the API? If so, I'll just change this to a switch on `paymentMethod`.